manifests/init.pp

```diff
@@ -90,7 +90,7 @@
   $java_opts                                                        = '',
   $catalina_opts                                                    = '',
   # Misc Settings
-  Variant[Stdlib::HTTPUrl, Stdlib::HTTPSUrl] $download_url          = 'https://downloads.atlassian.com/software/jira/downloads/',
+  Variant[Stdlib::HTTPUrl, Stdlib::HTTPSUrl] $download_url          = 'https://product-downloads.atlassian.com/software/jira/downloads/',
```

Comment:
I think we need to lose the trailing slash or looking at the travis failures we get a `403`.

Reply:
yes it causes double-slash -

`Error: Execution of '/usr/bin/curl https://downloads.atlassian.com/software/jira/downloads//atlassian-jira-software-7.13.1.tar.gz -o /tmp/atlassian-jira-software-7.13.1.tar.gz_20190223-8941-1h09xbl -fsSLg --max-redirs 5 --proxy ...' returned 22: curl: (22) The requested URL returned error: 404 Not Found`

Reply:
Odd that you got a `404` when the acceptance tests got a `403`
https://travis-ci.org/voxpupuli/puppet-jira/jobs/488570611#L1211
> Error: /Stage[main]/Jira::Install/Archive[/tmp/atlassian-jira-6.3.4a.tar.gz]/ensure: change from 'absent' to 'present' failed: Execution of '/opt/puppetlabs/puppet/bin/curl https://product-downloads.atlassian.com/software/jira/downloads//atlassian-jira-6.3.4a.tar.gz -o /tmp/atlassian-jira-6.3.4a.tar.gz_20190204-2261-jmy6j0 -fsSLg --max-redirs 5' returned 22: curl: (22) The requested URL returned error: 403 Forbidden

Either way, I think the double slashes are the problem.